meta-facebook/yv4-sd/src/platform/plat_init.c

```diff
@@ -108,6 +107,7 @@ void pal_set_sys_status()
 	sync_bmc_ready_pin();
 	set_sys_ready_pin(BIC_READY_R);
 	reset_usb_hub();
+	apml_init();
```

Comment:
@BonnieLo Can you please help us understand how this fixes the crash issue. Moving the apml_init() to pal_set_sys_status from pal_pre_init moves it to later stage?

Reply:
Hi @jagpalgill ,
We will check the length of SB-RMI command code in APML init function, and this action should do after we have set the status of POST complete.

Previously we did APML init before setting the status of POST complete which cause we set wrong length of SB-RMI command code in BIC.

Best Regards,
Sean